src/hyperfine/types.rs

```diff
@@ -157,3 +157,66 @@ impl BenchmarkResult {
         }
     }
 }
+
+/// Ensure that the default options are not changed unintentionally
+#[test]
+fn test_ensure_default_options_values() {
```

Comment:
To be honest, I'm not a big fan of this unit test. It is basically just re-iterating the default values from `types.rs` which means that we would have to change default values in two places. There is hardly any functionality that is tested here.

Reply:
If we were deriving Default, I'd agree. But that's the entire point, to prevent unintended changes to a known good application state. We know and have established what the default representation of the options should be. Preventing a regression, that could be hard to pinpoint, from being introduced is worth having to update the test when an explicit change is desired no?